src/components/form/widgets/input-field.ts

```diff
@@ -44,10 +44,7 @@ export class InputField extends React.Component {
             return;
         }
 
-        let value;
-        if (event.detail !== '') {
-            value = event.detail;
-        }
+        const value = event.detail || null;
```

Comment:
Hmm… using this, you can't empty any regular text-fields on the web client object card, or the save will fail, presumably because it doesn't expect null for string values… 😞

Errm… actually, testing a little further, _without_ this, the save won't fail, but it also won't save empty fields at all. It will just repopulate them with the old data…

I think what we need to do is output an empty string, unless the `type` is of a kind that doesn't return strings, in which case we should probably return `null` instead… 🤔